stetho/src/main/java/com/facebook/stetho/dumpapp/ArgsHelper.java

```diff
@@ -23,4 +25,12 @@ public static String nextArg(Iterator<String> argsIter, String errorIfMissing)
     }
     return argsIter.next();
   }
+
+  public static String[] argsIterToArray(Iterator<String> argsIter) {
```

Comment:
I just realized this method has a tragic side effect of exhausting the iterator as well which should also be captured in the name.  Prefer instead then `drainToArray` or something that communicates it has this side effect.